app/main.py

```diff
@@ -1 +1,23 @@
-# write your code here
+import shutil
+import os
+
+
+def move_file(command: str) -> any:
+    commands = command.split(" ")
+
+    if len(commands) == 3 and command.startswith("mv "):
+        _, first_file_name, path_to_second_file = commands
+        bare_path_to_second_file = os.path.dirname(path_to_second_file)
+        second_file_name = path_to_second_file.split("/")[-1]
```

Comment:
Is ur code cross-platform?) For example, will it work on win with \ ? Use os.path.split() instead

Reply:
If only I knew about os.path.split() yesterday, I would save a lot of time)